packages/local-serverless-server/README.md

```diff
@@ -0,0 +1,71 @@
+# @hubspot/local-serverless-server
+
+A server to test CMS serverless functions locally.
+
+## Getting started
+
+For more information on using these tools, see [Local Development Tooling: Getting Started](https://designers.hubspot.com/tutorials/getting-started-with-local-development)
+
+### Installation
+
```

Comment:
I'd suggest some docs that state that this is intended for use with the CLI with a link to either the docs on `developers.hubspot.com` or the NPM package.